mineinabyss-components/src/main/kotlin/com/mineinabyss/components/PlayerData.kt

```diff
@@ -18,7 +19,8 @@ class PlayerData(
     var orthCoinsHeld: Int = 0,
     var cloutTokensHeld: Int = 0,
     var showPlayerBalance: Boolean = true,
-    var guildChatStatus: Boolean = false
+    var guildChatStatus: Boolean = false,
+    var recentRightclickedEntity: Entity? = null
```

Comment:
Mark as `@Transient` (make sure it's the ktx.serialization one) since this will throw an error when it tries to look for a serializer for entity which doesn't exist. Alternatively, store as a UUID if you wish this to be persisted across relogs/restarts.